main.py

```diff
@@ -51,112 +27,233 @@ def send_static(path):
 db_port = os.getenv("DB_PORT", "5432")
 validateuser_url = os.getenv("VALIDATEUSER_URL", "http://localhost:5000")
 
-# connection pool config
-conn_pool_size = int(os.getenv("POOL_SIZE", "3"))
-conn_pool_max_overflow = int(os.getenv("POOL_MAX_OVERFLOW", "2"))
-conn_pool_timeout = float(os.getenv("POOL_TIMEOUT", "30.0"))
-conn_circuit_breaker = pybreaker.CircuitBreaker(
-    fail_max=1,
-    reset_timeout=10
-)
-
-
-@conn_circuit_breaker
-def create_conn():
-    conn = psycopg2.connect(host=db_host, database=db_name, user=db_user, password=db_pass, port=db_port)
-    return conn
-
+engine = create_engine("postgresql+psycopg2://" + db_user + ":" + db_pass + "@" + db_host +":"+ db_port + "/" + db_name)
+
+#adding custom details
+def custom_openapi():
+    if app.openapi_schema:
+        return app.openapi_schema
+    openapi_schema = get_openapi(
+        title=service_name,
+        version="",
+        description=service_name,
+        routes=app.routes,
+    )
+    openapi_schema["info"]["x-logo"] = {
+        "url": ""
+    }
+    app.openapi_schema = openapi_schema
+    return app.openapi_schema
 
-# connection pool init
-mypool = pool.QueuePool(create_conn, max_overflow=conn_pool_max_overflow, pool_size=conn_pool_size, timeout=conn_pool_timeout)
+app.openapi = custom_openapi
```

Comment:
pylint error - Cannot assigned to a method.  Do we need to use the custom openapi or can we get away with the default one?

Reply:
Just a nice to have feature for a more detailed render of API Doc. Removed it.